src/LaravelExactOnline.php

```diff
@@ -362,7 +362,7 @@ public function __call($method, $arguments)
             throw new RuntimeException("Invalid type called");
         }
 
-        return new $classname($this->connection());
+        return empty($arguments) ? new $classname($this->connection()) : new $classname($this->connection(), $arguments[0]);
```

Comment:
What would be the use case of only passing along the first argument?

Reply:
Picqer\Financials\Exact\Model's __construct method accepts second parameters as array which should be one dimensional array with attributes and values. 
For example if you call  $exactOnline->Account(['ID'=>'0000-....-0000']) the `$argument` will be two dimensional array and passing directly will not have any effect.